docs/src/Image.doc.js

```diff
@@ -38,6 +38,12 @@ card(
         required: true,
         href: 'placeholders',
       },
+      {
+        name: 'elementTiming',
+        type: 'string',
+        description:
+          'HTML attribute For performance profiling. See https://developer.mozilla.org/en-US/docs/Web/API/Element_timing_API',
```

Comment:
Do these need to be unique, specific strings to ensure accurate testing? Would be great to clarify that here

Reply:
@ayeshakmaz I want to keep the documentation as light as possible because this is for a very specific use case that requires reading the element timing API documentation